elastic/kibana
# Add license querystring to EMS requests in Vega

Fixes a regression in Vega Maps that limits the max zoom to 10 regardless of distribution.

## Summary

Fixes https://github.com/elastic/kibana/issues/112764

Vega Maps has a bug that limits the maxzoom for all distributions to 10. There are longer any license restrictions on EMS zoom levels in SSPL/ELv2 distributions. This PR adds a `?license=sspl` querystring to all HTTP requests to EMS to access zoom levels 11-18. 

Since this is a regression since 7.12 and the change here is small, I think we should consider patching this in 7.15.1.

To test:
1.  Load Sample Flight Data in Kibana
2. Open the "[Flights] Departures Count Map" Vega visualization
3. Zoom into a city until you see large scale features such as building footprints. This will indicate you are accessing zoom levels higher than 10.
